hw4_Zadacha7.py

```diff
@@ -0,0 +1,12 @@
+# Даны два натуральных числа. Вычислите их наибольший общий
+# делитель при помощи алгоритма Евклида (мы не знаем функции и рекурсию)
+
+num1 = int(input())
+num2 = int(input())
+while num1 != 0 and num2 != 0:
+    if num1 > num2:
+        num1 = num1 % num2
+    else:
+        num2 = num2 % num1
+
+print(num1 + num2)

```

Comment:
it's a hack. Algorithm doesn't expect result as a sum. Here you can just try not to detect 0 in one of your variables

Reply:
fixed?